doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -96,7 +96,7 @@ const Calendar = () => {
 
     return (
         <div className="PageLayout">
-            <div className={classnames("left-side-bar", theme ? "right-side-bg-blue" : "right-side-bg-green")} />
+            <div className={classnames("left-side-bar", theme ? "left-side-bg-blue" : "left-side-bg-green")} />
```

Comment:
?? Is this because on the calendar we do not have a right-side?

Reply:
right side is the wrong color, it was the lighter one, which we dont want obviously